src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,11 +149,41 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+
+    /*
+     *When collapsing expandCollapseButton, all panels buttons under it,
+     * (expandCollapseTeamButton and  expandCollapseStudentButton) should be in collapsed state.
+     * So that next time we expand the main panel, we find the expandCollapseTeamButton and
+     * expandCollapseStudentButton in collapsed state.
+     */
+    if (!panels && !isButtonInExpandMode) {
```

Comment:
This is really hard to understand. Extract out `const isCollapsePanelsButton = !panels`.